src/document/DocumentCommandHandlers.js

```diff
@@ -49,15 +50,18 @@ define(function (require, exports, module) {
         Strings             = require("strings"),
         PopUpManager        = require("widgets/PopUpManager"),
         PreferencesManager  = require("preferences/PreferencesManager"),
-        PerfUtils           = require("utils/PerfUtils"),
+        PerfUtils           = require("utils/PerfUtils"), 
         KeyEvent            = require("utils/KeyEvent"),
         Inspector           = require("LiveDevelopment/Inspector/Inspector"),
         Menus               = require("command/Menus"),
         UrlParams           = require("utils/UrlParams").UrlParams,
         StatusBar           = require("widgets/StatusBar"),
         WorkspaceManager    = require("view/WorkspaceManager"),
         LanguageManager     = require("language/LanguageManager"),
-        _                   = require("thirdparty/lodash");
+        _                   = require("thirdparty/lodash"),
+        CompressionUtils    = require("thirdparty/rawinflate"),
+        CompressionUtils    = require("thirdparty/rawdeflate"),
```

Comment:
Rename CompressionUtils with RawDeflate

Reply:
Hi @navch. Firstly, thank you for the quick reply! That aside, I am guessing you may not have seen my comment up above, but I'm having a unique problem when renaming to "RawDeflate" and trying to call RawDeflate.encode() or ...decode(). Hopefully you can help? I've never had this problem before. Require isn't working as expected.

Whenever I change variable name "CompressionUtils" to "RawDeflate", the Travis CI tests will pass, but Brackets stops working. When I run the code it says that "RawDeflate" is undefined in the console in dev tools when I call RawDeflate.encode() or .decode() as I was saying above. This doesn't make sense to me. Expected behavior is that if I require as RawDeflate, then RawDeflate.encode() or any other method should work.

However, when I change "RawDeflate" back to being literally any other variable name, Brackets will start to work again, but then the Travis CI tests fail because the variable names don't match. So this means even if I do ->
`CompressionUtils = require("thirdparty/rawinflate")`, I can't actually call CompressionUtils.inflate(). I still have to use "RawDeflate.inflate()", somehow, and somehow it works.

If you have a moment, can you please help?